invariant.ts

```diff
@@ -254,15 +277,27 @@ export const checkInvariants = (
           radio.emit(
             "logMessage",
             red(
-              `${invariantCallerWallet} [FAIL] ${getContractNameFromRendezvousId(
-                r.rendezvousContractId
-              )} ${underline(r.selectedInvariant.name)} ${printedInvariantArgs}`
+              `₿ ${simnet.burnBlockHeight.toString().padStart(6)} ` +
+                `Ӿ ${simnet.blockHeight.toString().padStart(6)}   ` +
+                `${invariantCallerWallet} ` +
+                `[FAIL] ` +
+                `${getContractNameFromRendezvousId(r.rendezvousContractId)} ` +
+                `${underline(r.selectedInvariant.name)} ` +
+                printedInvariantArgs
             )
           );
 
           // Re-throw the error for fast-check to catch and process.
           throw error;
         }
+
+        try {
+          if (r.mineBurnBlocks) {
+            simnet.mineEmptyBurnBlocks(r.burnBlocksToMine);
```

Comment:
Perhaps just `r.burnBlocks` would be easier, cleaner, and more action-oriented, suggesting blocks that are meant to be burned.